sourcegraph/cody
# change: enable new chat and search UIs by default

CONTEXT: https://sourcegraph.slack.com/archives/C066F85TE80/p1701709448915889

change: enable new chat and search UIs by default

- Enabled the new chat and search UIs by default by removing the `experimentalChatPanel` and `experimentalSearchPanel` configuration options.
- Remove Inline Chat and all related functions, commands and configuratoins (requested and confirmed by product: https://sourcegraph.slack.com/archives/C066F85TE80/p1701709918296539?thread_ts=1701709448.915889&channel=C066F85TE80&message_ts=1701709918.296539)
- Remove Inline Touch and Inline Chat recipes
- Updated all existing tests to run with new chat UI
- Remove old fixup tree view as part of the old chat UI (with old feature name)

## Follow up

Cean up chat manager and sidebar chat view.


## Test plan



All the tests have been updated to run on the new chat UI instead.

- confirm the new chat UI is enabled by default
- confirm inline-chat is no longer available
- confirm fixup tree view is no longer showing up
- will not be able to switch back to the old UI

<img width="774" alt="image" src="https://github.com/sourcegraph/cody/assets/68532117/abdf2795-4a67-46b0-b812-b72b01487c0e">
